prez/reference_data/profiles/prez_default_profiles.ttl

```diff
@@ -109,11 +107,14 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
                              skos:Collection ,
                              prez:CQLFilterResult ,
                              prez:QueryablesList ,
+                             sdo:DataCatalog ,
```

Comment:
Is this addition supposed to be in this PR?
Should there be a separate PR to add `sdo:DataCatalog`, `sdo:Dataset`, and `sdo:hasPart/isPartOf` support into prez default profiles?

Reply:
It was intentional for testing but I didn't think it through and will roll it back - otherwise I should roll out the SDO changes generally and I would prefer not to have the default endpoints/profiles cover all the alternate paths, it makes the endpoint definitions harder to read, and the queries more complex, and in most cases I think people should pick one or the other (if possible). I would rather put the effort into providing examples or making it easier for people to choose one or the other but not both.